python/ray/autoscaler/updater.py

```diff
@@ -36,7 +41,8 @@ def __init__(self,
                  runtime_hash,
                  process_runner=subprocess,
                  use_internal_ip=False,
-                 docker_config=None):
+                 docker_config=None,
+                 initialize_as_head=False):
```

Comment:
wait, so initialize_as_head is only used on the Docker command runner?

Reply:
Yes it is. I personally think that porting more of autoscaler to use it would be beneficial. It would mean that we don't have to do as much altering of the config.

Reply:
OK, got it - can you rename this to `is_head_node=True/False`